full_build.sh

```diff
@@ -20,6 +20,13 @@ rm -rf vendor
 composer clear-cache
 
 composer install --ansi
+
+# ensure clear cache
+bin/rector --version --clear-cache
+
+# ensure remove cache directory
+php -r 'shell_exec("rm -rf " . sys_get_temp_dir() . "/rector_cached_files");';
```

Comment:
this is really needed to ensure cleared data before run `full_build.sh` locally.